packages/css-library/src/stylesheets/uswds-typography.scss

```diff
@@ -0,0 +1,39 @@
+@forward 'uswds-theme';
+
+@import 'uswds-core/src/styles/mixins/typography/usa-content-styles';
+@import 'uswds-core/src/styles/mixins/typography/typeset';
+
+@import "uswds-fonts";
```

Comment:
This will load `@font-face` for "Source Sans Pro Web" with the file path coming from the USWDS fonts folder (as specific in the theme setting `theme-font-path`.